packages/dev/core/src/scene.ts

```diff
@@ -4956,6 +4956,10 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
         throw _WarnImport("Ray");
     }
 
+    public get _pickingAvailable(): boolean {

```

Comment:
```suggestion
    /** @internal */
    public get _pickingAvailable(): boolean {
```

Reply:
Could this be defined as a "static or var " in input manager instead ?